src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common.Tooling/LanguageServer/CSharpTestLspServerHelpers.cs

```diff
@@ -59,7 +59,7 @@ public static async Task<CSharpTestLspServer> CreateCSharpLspServerAsync(IEnumer
         var metadataReferences = (await ReferenceAssemblies.Default.ResolveAsync(language: LanguageNames.CSharp, cancellationToken))
             // ComponentBase here comes from our ComponentShim project, not the real ASP.NET libraries. It's enough for the generated C#
             // in tests to at least compile better.
-            .Add(MetadataReference.CreateFromFile(typeof(ComponentBase).Assembly.Location));
+            .Add(Basic.Reference.Assemblies.AspNet80.References.MicrosoftAspNetCoreComponents);
```

Comment:
Right now we're testing against AspNet80 because it's latest. At some point in the future we'd likel want to move to testing AspNet90. When that happens is the expectation that we'd do a big find / replace? Or did we want to centralize with a shared property somewhere that we could update?

Reply:
> Or did we want to centralize with a shared property somewhere that we could update?

Good idea, I will do that. Thanks.